### ISSUE TITLE
[Wasm|Skia]Unable to scroll a horizontable ListView

### ISSUE THREAD

**Issue by Xiaoy312** [unoplatform/uno#9847]
### Current behavior

cant scroll on a horizontal ListView with:
```xml
<ListView ItemsSource="123456789qweasdzxc"
          ScrollViewer.HorizontalScrollBarVisibility="Visible"
          ScrollViewer.HorizontalScrollMode="Enabled"
          ScrollViewer.VerticalScrollMode="Disabled">
	<ListView.ItemsPanel>
		<ItemsPanelTemplate>
			<ItemsStackPanel Orientation="Horizontal" />
		</ItemsPanelTemplate>
	</ListView.ItemsPanel>
</ListView>

```

### Expected behavior

should work like on uwp

### How to reproduce it (as minimally and precisely as possible)

https://playground.platform.uno/#590b892c

### Workaround

explicitly wrap it with a ScrollViewer

### Works on UWP/WinUI

Yes

### Environment

Uno.UI / Uno.UI.WebAssembly / Uno.UI.Skia

### NuGet package version(s)

latest

### Affected platforms

WebAssembly, Skia (GTK on Linux/macOS/Windows)

### IDE

Visual Studio 2022

### IDE version

n/a

### Relevant plugins

n/a

### Anything else we need to know?

n/a

**Comment by ramezgerges** [comment]
This was an issue with template binding and attached properties which is fixed by #13002 and I tested that it doesn't reproduce after the commit.

**Comment by MartinZikmund** [comment]
@ramezgerges Feel free to close if it is working now 👍
